bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx117-na-relay.html

```diff
@@ -0,0 +1,84 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ #}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}What’s new with Firefox{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+
+{% block page_css %}
+{{ css_bundle('firefox_whatsnew_117_na_relay') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block wnp_header %}
+<header class="c-page-header">
+  <div class="mzp-l-content c-page-header-inner">
+    <h2 class="c-page-header-logo-fx">{{ ftl('whatsnew-firefox') }}</h2>
+    {# aside is used to hide notification content from reader mode #}
+    <aside class="c-page-header-notification">
+      <div class="mzp-c-notification-bar mzp-t-success up-to-date">
+        <p>{{ ftl('whatsnew-up-to-date-notification-v2', fallback='whatsnew-up-to-date-notification') }}</p>
+      </div>
+      <div class="mzp-c-notification-bar out-of-date">
+        {% if ftl_has_messages('whatsnew-out-of-date-notification-v3') %}
+        <p>{{ ftl('whatsnew-out-of-date-notification-v3',
+          url='https://support.mozilla.org/kb/update-firefox-latest-release?' + utm_params) }}</p>
+        {% else %}
+        <p>{{ ftl('whatsnew-out-of-date-notification-v2') }}</p>
+        {% endif %}
+      </div>
+    </aside>
+  </div>
+</header>
+{% endblock %}
+
+{% block wnp_content %}
+<section class="wnp-content-main">
+  <div class="mzp-l-content mzp-t-content-sm">
+  <div class="wnp-body">
+    <div class="wnp-relay-animation">
+      <div class="relay-input">
+        <span>Enter your email address</span>
```

Comment:
I think adding `role="presentation"` will prevent this being read by screen readers? Or you could put this text in CSS with `content` to make it _really_ presentational.